configurations/es5-test.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    extends: [
+        '../configurations/es5.js',
```

Comment:
haha I was following the pattern `../` to be consistent and easy on the eye 😄 

this looks ugly 😛 
```js
extends: [
    './es5.js',
    '../rules/mocha/on.js'
],
```